src/utils/tokens/names.js

```diff
@@ -1,249 +1,20 @@
-      tokenName: 'Bonfida Token',
-      icon:
-        'https://raw.githubusercontent.com/dr497/awesome-serum-markets/master/icons/fida.svg',
-    },
-    {
-      tokenSymbol: 'LQID',
-      mintAddress: 'A6aY2ceogBz1VaXBxm1j2eJuNZMRqrWUAnKecrMH85zj',
-      tokenName: 'LQID',
-      icon:
-        'https://raw.githubusercontent.com/dr497/awesome-serum-markets/master/icons/lqid.svg',
-    },
-    {
-      tokenSymbol: 'SECO',
-      mintAddress: '7CnFGR9mZWyAtWxPcVuTewpyC3A3MDW4nLsu5NY6PDbd',
-      tokenName: 'Serum Ecosystem Pool Token',
-    },
-    {
-      tokenSymbol: 'HOLY',
-      mintAddress: '3GECTP7H4Tww3w8jEPJCJtXUtXxiZty31S9szs84CcwQ',
-      tokenName: 'Holy Trinity Pool',
-    },
-    {
-      tokenSymbol: 'KIN',
-      mintAddress: 'kinXdEcpDQeHPEuQnqmUgtYykqKGVFq6CeVX5iAHJq6',
-      tokenName: 'KIN',
-      icon:
-        'https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/kin/info/logo.png',
-    },
-    {
-      tokenSymbol: 'MAPS',
-      mintAddress: 'MAPS41MDahZ9QdKXhVa4dWB9RuyfV4XqhyAZ8XcYepb',
-      tokenName: 'Maps.me Token',
-    },
-    {
-      tokenSymbol: 'RAY',
-      mintAddress: '4k3Dyjzvzp8eMZWUXbBCjEvwSkkk59S5iCNLY3QrkX6R',
-      tokenName: 'Raydium',
-      icon:
-        'https://raw.githubusercontent.com/raydium-io/media-assets/master/logo.svg',
-    },
-    {
-      tokenSymbol: 'RAY-USDT',
-      mintAddress: 'CzPDyvotTcxNqtPne32yUiEVQ6jk42HZi1Y3hUu7qf7f',
-      tokenName: 'Raydium USDT Liquidity Pool',
-      icon:
-        'https://raw.githubusercontent.com/raydium-io/media-assets/master/logo.svg',
-    },
-    {
-      tokenSymbol: 'RAY-USDC',
-      mintAddress: 'FgmBnsF5Qrnv8X9bomQfEtQTQjNNiBCWRKGpzPnE5BDg',
-      tokenName: 'Raydium USDC Liquidity Pool',
-      icon:
-        'https://raw.githubusercontent.com/raydium-io/media-assets/master/logo.svg',
-    },
-    {
-      tokenSymbol: 'RAY-SRM',
-      mintAddress: '5QXBMXuCL7zfAk39jEVVEvcrz1AvBGgT9wAhLLHLyyUJ',
-      tokenName: 'Raydium Serum Liquidity Pool',
-      icon:
-        'https://raw.githubusercontent.com/raydium-io/media-assets/master/logo.svg',
-    },
-    {
-      tokenSymbol: 'OXY',
-      mintAddress: 'z3dn17yLaGMKffVogeFHQ9zWVcXgqgf3PQnDsNs2g6M',
-      tokenName: 'Oxygen Protocol',
-      icon:
-        'https://raw.githubusercontent.com/nathanielparke/awesome-serum-markets/master/icons/oxy.svg'
-    },
-  ],
-};
+const tokenListProvider = new TokenListProvider();
+export function useTokenInfos() {
+  const [tokenListContainer] = useAsyncData(tokenListProvider.resolve, tokenListProvider.resolve, { refreshInterval: 120000});
+  const { endpoint } = useConnectionConfig();
+  const cluster = clusterForEndpoint(endpoint);
+
+  const filteredTokenListContainer = tokenListContainer?.filterByClusterSlug(cluster?.name);
+  return tokenListContainer !== filteredTokenListContainer ? filteredTokenListContainer?.getList(): null; // Workaround for filter return all on unknown slug
```

Comment:
If you want to filter NFTs you can use `.excludeByTag("nft")`

Reply:
Do we want them excluded?